osu.Game/Beatmaps/Drawables/StarRatingDisplay.cs

```diff
@@ -4,12 +4,12 @@
 using osu.Framework.Allocation;
 using osu.Framework.Bindables;
 using osu.Framework.Extensions.Color4Extensions;
+using osu.Framework.Extensions.LocalisationExtensions;
```

Comment:
I'm not super thrilled about this. Almost feels like it should be in `osu.Framework.Localisation` as it was previously.

But I guess it's early enough in development that it's not a big deal...?

Reply:
Do you mean "extensions" appearing twice in the full namespace? There are lots more cases of this in framework. One directly above, even. Was mentioned in the past, even, but not acted upon.

Reply:
Just existing in the `Extensions` namespace at all.

Reply:
If it's because the localisation code is an osu!framework thing, the same would apply to `osu.Framework.Extensions.PolygonExtensions.(Convex)PolygonExtensions`, `osu.Framework.Extensions.AnchorExtensions` and `osu.Framework.Extensions.PopoverExtensions`, which all receive framework types as input.

Reply:
More like because it's a frequently-used thing. 30+ files were changed in this PR alone as a result of this change.